Mage/src/main/java/mage/target/common/TargetActivatedAbility.java

```diff
@@ -79,9 +79,9 @@ public Set<UUID> possibleTargets(UUID sourceControllerId, Ability source, Game g
     public Set<UUID> possibleTargets(UUID sourceControllerId, Game game) {
         Set<UUID> possibleTargets = new HashSet<>();
         for (StackObject stackObject : game.getStack()) {
-            if (stackObject.getStackAbility().getAbilityType() == AbilityType.ACTIVATED
+            if (stackObject.getStackAbility().isActivatedAbility()
                     && game.getState().getPlayersInRange(sourceControllerId, game).contains(stackObject.getStackAbility().getControllerId())
-                    && filter.match(stackObject, game)) {
+                    && filter.match(stackObject, game)){
```

Comment:
(why remove this space, our standard style includes it)

Reply:
Uhm weird. May need to tweak my auto formatter settings in intelliJ.